annotated-projects/MTBLS1903/MTBLS1903.sdrf.tsv

```diff
@@ -0,0 +1,136 @@
+source name	characteristics[organism]	characteristics[organism part]	characteristics[sex]	characteristics[biological replicate]	characteristics[disease]	characteristics[cell type]	assay name	technology type	comment[technical replicate]	comment[chromatography instrument]	comment[chromatography type]	comment[scan polarity]	comment[instrument]	comment[ion source]	comment[data file]	comment[metabolite assignment file]	comment[fraction identifier]	factor value[gene]

```

Comment:
Hi! I agree, as discussed in the issue, fractionation is rare, so I'll remove it, I just included potentially interesting columns here to continue the discussion. And you're also right on the  `chromatography instrument`, that shouldn't be necessary. `chromatography type` is very important in my opinion since this directly impacts downstream processing, same as the `scan polarity`. Specifying the `instrument` is debatable but at least the instrument type should be specified. Similar for the `ion source`, although this could be omitted. I would keep the `metabolite assignment file` since most types of analysis will make use of one and having this information is critical to recreate the results

Reply:
@mmattano @nilshoffmann (also can give input here): For a specification of SDRF metabolomics, we first need to define the requirement columns (which I, sometime, called the minimum information needed to represent the experiment). 

In this, PR can try to define what will be the minimum info for the metabolomics dataset. BTW, I do think the sample metadata (characteristics it is the same between both metabolomics and proteomics), the main work should be done in the data site comments.